threads.php

```diff
@@ -135,12 +135,14 @@ function cfth_thread_notice($posts, $query) {
 
 function cfth_timeline_posts($term_id) {
 	$term = get_term_by('id', $term_id, 'threads');
+	$term_meta = get_term_meta( $term_id, 'threads_meta', true );
+	$order = ( isset( $term_meta['order_term_meta'] )) ? $term_meta['order_term_meta'] : 'ASC';
```

Comment:
there is an extra set of parenthesis around the first item in your ternary operation here, you can remove these